application/models/User_model.php

```diff
@@ -536,9 +536,10 @@ function authorize($level) {
 			$level = $this->config->item('auth_mode');
 		}
 		if(($this->validate_session($u)) && ($u->row()->user_type >= $level) || $this->config->item('use_auth') == FALSE || $level == 0) {
-			$ls=new DateTime($u->row()->last_seen ?? '1971-01-01');
-			$n=new DateTime("now");
-			if ($ls->diff($n)->format('%s') > 60) {	// Reduce load of the Spy-Function. shouldn't be called at anytimne. 60seconds diff is enough
+			$ls = strtotime($u->row()->last_seen ?? '1971-01-01');
```

Comment:
Is there a reason we use 1971 instead of 1970 (i.e. Unix timestamp of 0).

Reply:
So shall I fix it (directly)?

Reply:
yep.

Reply:
Fixed.